### DISCUSSION TITLE
api-delete bug, help

### DISCUSSION THREAD

**Discussion by ouyoung33** [jaredhendrickson13/pfsense-api#434] [General]
After upgrading to API-v2, I want to go back to v1, directly delete the api, prompt instruction can not be found, and then I restore the backup of the entire firewall, there are api options in the menu, it is 403, I delete again or prompt instruction can not be found, reinstall also can not

**Comment by ouyoung33** [comment]
![12](https://github.com/jaredhendrickson13/pfsense-api/assets/128574646/6b480a4f-430d-444c-b549-c4103e014293)
![123](https://github.com/jaredhendrickson13/pfsense-api/assets/128574646/47b25637-70fc-4b70-b3b8-a234a10af8d2)

**Comment by ouyoung33** [comment]
![Uploading 1.png…]()

**Comment by jaredhendrickson13** [comment]
The command to delete v2 is `pfsense-restapi delete` or `pkg-static delete pfSense-pkg-RESTAPI`. Seems like you may be looking at documentation on the master branch, which still contains the codebase for v1. The codebase for v2 is currently operating off the `next_major` branch as it is still in development. You can refer to [this discussion](https://github.com/jaredhendrickson13/pfsense-api/discussions/430) for important information about the v2 development builds.

**Reply by ouyoung33** [reply]
So my best bet right now is to reinstall it? Because the API-delete command can no longer be recognized, but the gui interface can now see the api options, click on it is reported 403

**Reply by jaredhendrickson13** [reply]
I'll need the output of the commands I mentioned above to troubleshoot further and tell for sure.

Can you provide the output of `pfsense-restapi delete`? If that gives an error, can you also give the output of `pkg-static delete pfSense-pkg-RESTAPI`?
